code/modules/antagonists/heretic/heretic_living_heart.dm

```diff
@@ -108,12 +108,25 @@
 /datum/action/cooldown/track_target/Activate(atom/target)
 	var/datum/antagonist/heretic/heretic_datum = GET_HERETIC(owner)
 	var/datum/heretic_knowledge/sac_knowledge = heretic_datum.get_knowledge(/datum/heretic_knowledge/hunt_and_sacrifice)
+
 	if(!LAZYLEN(heretic_datum.sac_targets))
 		owner.balloon_alert(owner, "no targets, visit a rune!")
 		StartCooldown(1 SECONDS)
 		return TRUE
 
 	var/list/targets_to_choose = list()
+
+	var/obj/item/melee/sickly_blade/blade
+	for(var/datum/weakref/blade_ref as anything in heretic_datum.created_blades)
+		blade = blade_ref.resolve()
```

Comment:
Why is `blade` being tracked outside of the loop?

Doesn't that imply it'll pretty much always be set if you have a missing blade?